tools/js-sdk-release-tools/docs/llc.md

```diff
@@ -0,0 +1,82 @@
+# Azure JS RLC Tool
+Azure JS RLC Tool can help you get llc codes by one command in few minutes.
+
+*Note: Currently, this tools only supports generating dataplane sdk, and only can run in powershell/bash.*
+
+## Install
+```shell script
+npm install -g @azure-tools/js-sdk-release-tools
+```
+
+## How to use
+
+### Prerequisites
+1. Install Any of the [LTS versions of Node.js](https://nodejs.org/en/about/releases/).
+2. Install autorest: `npm install -g autorest`.
+3. Install rush: `npm install -g @microsoft/rush`.
+4. use [Git](https://git-scm.com/) to clone [azure-sdk-for-js](https://github.com/Azure/azure-sdk-for-js) or [azure-sdk-for-js-pr](https://github.com/Azure/azure-sdk-for-js-pr). Then go to the root of sdk repository and run `rush update`. For example:
+```shell
+cd azure-sdk-for-js
+rush update
+```
+### Steps
+
+##### Step 1. Run command:
+```
+rlc-code-gen --package-name=<your package name>
+```
+*Note*:
+   - *Please replace with your package name, for example: `rlc-code-gen --package-name=@azure-rest/purview-account`. If you don't provide it, the tool will ask you to input it interactively.*
+   - *You also can use command `rlc-code-gen --readme=<the path of swagger/README.md>` to generate codes if `swagger/README.md` exists. Then it will skip Step 2 and go to Step 3 directly.*
+##### Step 2. Generate swagger/README.md
+If your package is first release or doesn't contain `swagger/readme.md`, the tool will ask you to input some necessary information. If not, please skip this step.
+   
+For example:
+
+```
+Which resource provider do you want to store your package in sdk folder? Please input it: purview
+```
+
+You need to provide following information. You can refer to [README.md](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/purview/purview-catalog-rest/swagger/README.md) as an example.
+
+| parameter | description |
+| ----- | ---- |
+| resource-provider | The folder name under [sdk](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk). If the resource provider name you provided is not under [sdk](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk), we will create a new one for you. |
```

Comment:
I think call it resource-provider can be misleading, because service team would think it as Microsoft.XXX. Maybe just parent folder ?

Reply:
I think the description tells the customer it's under sdk folder.

Reply:
they may think it's sdk/Microsoft.Compute. purely service name is enough.